### PR TITLE
Issue 187 - change parameters from camelCase to hyphen-names

### LINKED ISSUE #187: Switch to dashed case from the camelCase in sti CLI
Since origin also now use dashed syntax, we should switch our options as well, so instead of `--forcePull` just `--force-pull`. Same for other options...

@bparees @soltysh should be easy fix and should be done before we release 'sti' standalone (or somebody package it ;-)

### PR DESCRIPTION
**Pull request by soltysh**
Fixes #187.
@bparees @mfojtik PTAL

### CODE REVIEW

File: cmd/sti/main.go [hunk 1]
```diff
@@ -128,14 +128,14 @@ func newCmdBuild(req *api.Request) *cobra.Command {
...
-	buildCmd.Flags().BoolVar(&(req.PreserveWorkingDir), "saveTempDir", false, "Save the temporary directory used by STI instead of deleting it")
+	buildCmd.Flags().BoolVar(&(req.ForcePull), "force-pull", true, "Always pull the builder image even if it is present locally")
+	buildCmd.Flags().BoolVar(&(req.PreserveWorkingDir), "save-temp-dir", false, "Save the temporary directory used by STI instead of deleting it")
 	buildCmd.Flags().BoolVar(&(useConfig), "use-config", false, "Store command line options to .stifile")
-	buildCmd.Flags().StringVarP(&(req.ContextDir), "contextDir", "", "", "Specify the sub-directory inside the repository with the application sources")
-	buildCmd.Flags().StringVarP(&(req.DockerCfgPath), "dockerCfgPath", "", filepath.Join(os.Getenv("HOME"), ".dockercfg"), "Specify the path to the Docker configuration file")
+	buildCmd.Flags().StringVarP(&(req.ContextDir), "context-dir", "", "", "Specify the sub-directory inside the repository with the application sources")
+	buildCmd.Flags().StringVarP(&(req.DockerCfgPath), "docker-cfg-path", "", filepath.Join(os.Getenv("HOME"), ".dockercfg"), "Specify the path to the Docker configuration file")
```
Comment (mfojtik) [inline cmd/sti/main.go]: dockercfg-path

File: docs/cli.md [hunk 2]
```diff
@@ -63,22 +63,22 @@ that image and add them to the tar streamed to the container into `/artifacts`.
...
+| `--force-pull`              | Always pull the builder image, even if it is present locally |
 | `-l (--location)`          | Location where the scripts and sources will be placed prior doing build (see [STI Scripts](#sti-scripts))|
 | `-r (--ref)`               | A branch/tag from which the build should happen (applies only to GIT source) |
 | `--rm`                     | Remove previous image during incremental build |
-| `--saveTempDir`            | Save the working directory used for fetching scripts and sources |
-| `--contextDir`             | Allow to specify directory name with your application |
+| `--save-temp-dir`            | Save the working directory used for fetching scripts and sources |
+| `--context-dir`             | Allow to specify directory name with your application |
 | `-s (--scripts)`           | URL of STI scripts (see [STI Scripts](#sti-scripts)) |
 | `-q (--quiet)`             | Operate quietly, suppressing all non-error output |
```
Comment (mfojtik) [inline docs/cli.md]: can you please add --dockercfg-path here (and check if there is something missing ;-) I know, I know, my fault, I should add that ;-)

**Comment by mfojtik** [comment]
LGTM \o/ (one small naming suggestion as 'dockercfg' is the name of the file ;-)

**Comment by soltysh** [comment]
@mfojtik done.

**Comment by mfojtik** [comment]
@soltysh will wait for Ben to see this before merge ;-)

**Comment by bparees** [comment]
lgtm [merge]

**Pull request merged**